chapter14/README.md

```diff
@@ -1,17 +1,54 @@
-# Chapter 13
+Run Instructions
 
-Code accompanying Chapter 13 (Securing Airflow) of the book [Data Pipelines with Apache Airflow](https://www.manning.com/books/data-pipelines-with-apache-airflow).
+# Run Instructions
 
-## Contents
+0) Get into the project directory
+    `cd chapter13_genai`
 
-This folder holds three example Docker Compose examples:
 
-- `ldap`: Example configuration of the webserver, fetching user credentials from OpenLDAP
-- `rbac`: Example running the RBAC interface
-- `secretsbackend`: Example fetching secrets from HashiCorp Vault
+1) Run the Docker compose file
+    `docker-compose up`
 
-Each folder holds a Docker Compose file which can be started with `docker-compose up -d`.
+2) Once MinIO is running, you can access the MinIO web interface at `http://localhost:8083` with the following credentials:
+    - Access Key: `airflow`
+    - Pass: `apacheairflow`
 
-## Usage
+3) Copy all the contents of the `.env.template` file in the root of the chapter folder to the `.env` file
 
-Read the `README.md` file in the respective directory.
+4) Create an access key in MinIO UI and update the `.env` file in this repo on the following variables
```

Comment:
Can;t we just setup minio already with an key and secret so the user does not have to do it?